frontend/src/services/user.ts

```diff
@@ -1,39 +1,22 @@
 import {User, UserCredentials, UserType} from "../types/user";
+import httpClient from "./httpClient";
+import {HttpPaths} from "../common/constansts/paths";
+import _ from "lodash";
 
 export const getUserService = async (userId: number, userType: UserType): Promise<User> => {
     if (userType === UserType.Customer) {
-        return {
-            id: userId,
-            email: "user@gmail.com",
-            name: "User",
-            createdAt: new Date(),
-            updatedAt: new Date(),
-        }
-    }
-    return {
-        id: userId,
-        email: "usermerchant@gmail.com",
-        name: "UserMerchant",
-        createdAt: new Date(),
-        updatedAt: new Date(),
+        const response = await httpClient.get<User>(`${HttpPaths.CUSTOMER}/${userId}`);
+        return _.get(response, "data");
     }
+    const response = await httpClient.get<User>(`${HttpPaths.MERCHANT}/${userId}`);
+    return _.get(response, "data");
```

Comment:
you can extract "data" to a constant or move this line to utils file and reuse it everywhere.

Reply:
What about this comment ?

Reply:
in this case, idk how i get data from logInService(), because i extract data: "data.token" and for some cases i don't need to return default value or return default value based on type (null, empty string or smt else)